addons/gopaddle-lite/enable

```diff
@@ -0,0 +1,188 @@
+  
+  if [ "$NODELABEL" = "0" ]
+  then
+    #getting node name
+    NODENAME=$(run_with_sudo $KUBECTL get node -o json| jq '.items[0].metadata.name'|cut -d '"' -f 2)
+    echo ""
+    echo "Adding label 'gp-install-node=node1' to the node '$NODENAME'"
+    echo ""
+    #adding label in node 
+    run_with_sudo $KUBECTL label nodes $NODENAME gp-install-node=node1 --overwrite=true
+  else
+    #getting node name
+    NODENAME=$(run_with_sudo $KUBECTL get node -l gp-install-node=node1 -o json| jq '.items[0].metadata.name'|cut -d '"' -f 2)
+  fi
+
+  if [ $PUBLICIP == true ]
+  then
+    #install rabbitmq
+    run_with_sudo $HELM install gp-rabbitmq-4-2 gp-lite/gp-lite  --namespace gp-lite  --set global.routingType=NodePortWithOutIngress --set global.installType=public --set global.storageClassName=$STORAGECLASS --set global.gp-rabbitmq.enabled=true --set global.gp-lite-core.enabled=false --set global.nodeSelector.gp-install-node=node1 --set global.nodeName=$NODENAME  --version $HELM_VERSION
+
+    #install gp-lite
+    run_with_sudo $HELM install gp-core-4-2 gp-lite/gp-lite  --namespace gp-lite  --set global.routingType=NodePortWithOutIngress --set global.installType=public --set global.storageClassName=$STORAGECLASS --set global.cluster.provider=other --set-string global.gopaddle.https=false --set-string global.gopaddleWebhook.https=false --set global.staticIP=$STATICIP  --set global.gp-rabbitmq.enabled=false --set global.gp-lite-core.enabled=true --set global.installSource=microk8s --set global.nodeSelector.gp-install-node=node1 --set global.nodeName=$NODENAME  --version $HELM_VERSION
+  elif [ $PUBLICIP == false ]
+  then 
+    #install rabbitmq
+    run_with_sudo $HELM install gp-rabbitmq-4-2 gp-lite/gp-lite  --namespace gp-lite  --set global.routingType=NodePortWithOutIngress --set global.installType=private --set global.storageClassName=$STORAGECLASS --set global.gp-rabbitmq.enabled=true --set global.gp-lite-core.enabled=false  --set global.nodeSelector.gp-install-node=node1 --set global.nodeName=$NODENAME --version $HELM_VERSION
+
+    #install gp-lite
+    run_with_sudo $HELM install gp-core-4-2 gp-lite/gp-lite  --namespace gp-lite  --set global.routingType=NodePortWithOutIngress --set global.installType=private --set global.storageClassName=$STORAGECLASS --set global.cluster.provider=other --set-string global.gopaddle.https=false --set-string global.gopaddleWebhook.https=false --set global.staticIP=$STATICIP  --set global.gp-rabbitmq.enabled=false --set global.gp-lite-core.enabled=true --set global.installSource=microk8s --set global.nodeSelector.gp-install-node=node1 --set global.nodeName=$NODENAME --version $HELM_VERSION
+  fi
+
+  echo ""
+  echo "Waiting for the gopaddle volume to move bound. This may take a while."
+  while true
+  do
+    if microk8s kubectl wait pvc --for=jsonpath='{.status.phase}'=Bound -l released-by=gopaddle -n gp-lite --timeout=20m 2>/dev/null; then
```

Comment:
Why not use $KUBECTL here as well?

Reply:
I have verified the usage of $KUBECTL across the script. I have made sure it is consistent throughout the script.